### PR TITLE
Add Future Times Three

### PR DESCRIPTION
**Pull request by gibet**
An amazing book about time travels by René Barjavel. Original title : "Le voyageur imprudent"

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Awesome Sci-Fi [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+﻿# Awesome Sci-Fi [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
```
Comment (RichardLitt) [inline readme.md]: Why was this line committed? :?

File: readme.md [hunk 2]
```diff
@@ -827,6 +827,12 @@ I liked this series so much I got a tattoo partially inspired by it. - [@Richard
 
 > They never imagine that the future time in which Dan awakens has a very limited form of time travel, just enough that Davis can travel back and recover his research. He then again undergoes suspended animation, and awakens again in the high-tech future of the year 2000, with his reputation, fortune, and his sweetheart.
 
+#### [Future Times Three](https://www.goodreads.com/book/show/2509242.Future_Times_Three) (1968) *by [René Barjavel](https://en.wikipedia.org/wiki/Ren%C3%A9_Barjavel)* [3.8]
+
+A really good story about time travels, their consequences and the famous [Grandfather paradox](https://en.wikipedia.org/wiki/Grandfather_paradox). [@Gibet](https://github.com/Gibet)
```
Comment (RichardLitt) [inline readme.md]: Add a dash between `.` and `[@Gibet`.

Comment (gibet) [inline readme.md]: Done, thanks

File: readme.md (hunk 1 above)
Comment (gibet) [inline readme.md]: I don't know... and I don't know how to revert... nothing ? :o

**Comment by RichardLitt** [comment]
Well, LGTM! Thank you!

**Comment by RichardLitt** [comment]
And welcome! Glad to have your first PR here. ❤️

**Pull request merged**